containers/ruby/definition-manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"variants": ["3.1-bullseye", "3.0-bullseye", "2.7-bullseye", "2.6-bullseye", "3.1-buster", "3.0-buster", "2.7-buster", "2.6-buster"],
+	"variants": ["3.1-bullseye", "3.0-bullseye", "2.7-bullseye", "3.1-buster", "3.0-buster", "2.7-buster"],
 	"definitionVersion": "0.203.2",
```

Comment:
we should also bump the minor `definitionVersion` here to trigger a new release of this image.

Reply:
Bumped the version - let me know if this looks good to you @joshspicer.